src/components/SingleArticle.jsx

```diff
@@ -2,6 +2,7 @@ import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { articleById } from "../utils/api";
```

Comment:
Just needs some handling of errors. At the moment, if i go to an article id that isnt there, it just constantly says loading. 
needs a little 404 to tell us we've gone wrong